<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>871</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(195, 190, 180);
border-radius:30px
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgba(0,0,0,210);
</string>
      </property>
      <widget class="QWidget" name="horizontalWidget_5" native="true">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>0</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>141</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(255,255,255,255)</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;INVENTARIO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalWidget_6" native="true">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>0</y>
         <width>51</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:rgba(0,0,0,0);
	color:rgb(255,255,255);
	border-radius:1px;
	font-size:18px;
	font-family:dripicons-v2;
}

QPushButton:hover{
	background-color:rgb(49,48,53)
}

QPushButton#cerrar:hover{
	background-color:rgb(232,17,25)
}

QpushButton:pressed{
	padding-top:5px;
 	padding-left:5px;
}
</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnCerrar">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>871</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);
border-radius:30px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblMaterial">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>611</width>
        <height>421</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
border: 3px solid  #000000;
background-color: rgb(199, 194, 184);
</string>
      </property>
      <column>
       <property name="text">
        <string>Id-Material</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Id-Servicio</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descripcion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stock</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Medicion</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frente</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/newPrefix/Marohuma_icono.png);
background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Imagenes/QT_imagenes.qrc">:/newPrefix/Marohuma_icono.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cboListar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(195, 190, 180);
border:1.5px solid #7c6e59;
border-radius:10px;
color:rgba(0,0,0,240);
padding-bottom:4px;
padding-left:4px</string>
      </property>
      <item>
       <property name="text">
        <string>--Todos--</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(228, 222, 211);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Listar:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>widget_3</zorder>
     <zorder>tblMaterial</zorder>
     <zorder>label_6</zorder>
     <zorder>label</zorder>
     <zorder>cboListar</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes/QT_imagenes.qrc"/>
  <include location="Imagenes/QT_imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
